### PR TITLE
addressed Actor model post feedback

### PR DESCRIPTION
**Pull request by gavination**

### CODE REVIEW

**Comment by laurakalbag** [comment]
@gavination I’ve updated your post with a shiny social media image and updated the frontmatter so you have your author pic! Also fixed a link. If you want to give it a quick once-over, we can publish!

File: blog/2023-5-30-what-is-the-actor-model-and-when-should-i-use-it/index.mdx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      }),
+    },
+    // ...
+  },
+});
+```
+
+With the use of `spawn()` and `assign()`, we create a new actor instance when provided the machine logic and a unique identifier.
+By their nature, actions are ["fire and forget" effects](https://xstate.js.org/docs/guides/actions.html), meaning they are executed with no expectation of receiving events back to the actor. This makes sense for creating a new actor, but we may still want the parent actor to have a reference to its child, so we save that in its context using `assign()`. `spawn()` is the function called that actually creates the new actor. The parent can access this state easily by calling `getSnapshot()` on the reference to the child.
+For more detailed examples around working with actors in XState, like callback or promised-based actor spawning, sending updates, and communicating between actors, [check out our guides](https://xstate.js.org/docs/guides/actors.html).
```
Comment (laurakalbag) [inline blog/2023-5-30-what-is-the-actor-model-and-when-should-i-use-it/index.mdx:104]:
```suggestion

For more detailed examples around working with actors in XState, like callback or promised-based actor spawning, sending updates, and communicating between actors, [check out our docs on actors](actors.mdx).
```

1. Put a line between this paragraph and the text above to break the text up and make it read more as a conclusion.
2. Update the link to reference our new docs
3. Make the link text a little more descriptive

File: blog/2023-5-30-what-is-the-actor-model-and-when-should-i-use-it/index.mdx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        ],
+      }),
+    },
+    // ...
+  },
+});
+```
+
+With the use of `spawn()` and `assign()`, we create a new actor instance when provided the machine logic and a unique identifier.
+By their nature, actions are ["fire and forget" effects](https://xstate.js.org/docs/guides/actions.html), meaning they are executed with no expectation of receiving events back to the actor. This makes sense for creating a new actor, but we may still want the parent actor to have a reference to its child, so we save that in its context using `assign()`. `spawn()` is the function called that actually creates the new actor. The parent can access this state easily by calling `getSnapshot()` on the reference to the child.
```
Comment (laurakalbag) [inline blog/2023-5-30-what-is-the-actor-model-and-when-should-i-use-it/index.mdx:103]:
```suggestion
By their nature, actions are ["fire and forget" effects](actions.mdx), meaning they are executed with no expectation of receiving events back to the actor. This makes sense for creating a new actor, but we may still want the parent actor to have a reference to its child, so we save that in its context using `assign()`. `spawn()` is the function called that actually creates the new actor. The parent can access this state easily by calling `getSnapshot()` on the reference to the child.
```

Comment (gavination) [reply blog/2023-5-30-what-is-the-actor-model-and-when-should-i-use-it/index.mdx]: LOVE directly linking to our own docs in the repo!

**Review by laurakalbag — APPROVED** [review]
Hooray! Can’t wait to share this! Please feel free to merge.

**Pull request merged**